platform/mbed_lib.json

```diff
@@ -37,8 +37,8 @@
         },
 
         "force-non-copyable-error": {
-            "help": "Force compile time error when a NonCopyable object is copied",
-            "value": false
+            "help": "Force compile time error when a NonCopyable object is copied (obsolete option - always treated as true)",
+            "value": true
```

Comment:
Why not remove the option since it is no longer used?

Reply:
I was being cautious about breaking apps - removing an option is an immediate break for any code setting it, regardless of what they were setting it to.

But I guess if this is labelled as a breaking change anyway, may as well take it out?

Reply:
Removed it.